tests/openvino/test_seq2seq.py

```diff
@@ -837,6 +847,9 @@ def test_generate_utils(self, model_arch):
             input_audio = self._generate_random_audio_data()
             question = "Translate this audio to French"
             inputs = model.preprocess_inputs(**preprocessors, text=question, audio=[input_audio])
+            # skip the temporal_ids which makes the number of loop inconstant:
+            # https://huggingface.co/openbmb/MiniCPM-V-4_5/blob/main/resampler.py#L261
+            inputs.pop("temporal_ids", None)
```

Comment:
don't quite understand. To me it degrades user experience that we make users to skip this parameter after preprocessing to have matched results with the reference.

Reply:
still not fixed